README.md

```diff
@@ -1,2 +1,51 @@
 # playbook
-collection of venturemark playbooks
+
+Collection of playbooks for the Venturemark platform.
+
+
+
+### What Is A Playbook?
+
+A playbook aims to provide you with inspiration and instruction for how to use
+the Venturemark platform. The Venturemark platform provides you with an abstract
+and powerful way to combine contextual information and factual data. Due to the
+abstract nature of the Venturemark platform, you can use it in many different
+ways. There are many powerful use cases for high performing individuals to use
+the Venturemark platform. A playbook is merely a piece of documentation
+providing you with instructions for habit forming processes. These processes aim
+to achieve a track record of better results throughout your venture's journey.
+
+
```

Comment:
If the purpose is outward facing rather than internal -> more concise: A playbook is a way you can use Venturemark to benchmark and track progress.